packages/mc-scripts/src/commands/login.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path');
+const { promisify } = require('util');
+const read = promisify(require('read'));
+const fetch = require('node-fetch');
+const fs = require('fs');
+const homedir = require('os').homedir();
+const { processConfig } = require('@commercetools-frontend/application-config');
+const { MC_API_URLS, DEFAULT_CREDENTIALS } = require('../constants');
+
+function validateEmail(email) {
+  const re = /\S+@\S+\.\S+/;
+  return re.test(email);
+}
+
+const getEmail = async () => {
+  let email = await read({ prompt: 'Email: ' });
+  if (!validateEmail(email)) {
+    console.log('Please input a valid email');
+    await getEmail();
```

Comment:
This is looking good 🙌 One suggestion:
```suggestion
    email = await getEmail();
```